ui/src/onboarding/components/CompletionAdvancedButton.tsx

```diff
@@ -31,9 +31,13 @@ class CompletionAdvancedButton extends PureComponent<Props> {
 
   private handleAdvanced = (): void => {
     const {router, orgs} = this.props
-    const id = _.get(orgs, '[0].id', null)
+    const id = _.get(orgs, '0.id', null)
+
     if (id) {
-      router.push(`/organizations/${id}/buckets_tab`)
+      router.push({
+        pathname: `/organizations/${id}/buckets_tab`,
+        state: {openDataLoaderOverlay: true},
```

Comment:
we could allow a user to go to the buckets overlay using a url like 

```
/organizations/1/buckets_tab?openDataLoaderOverlay=true
```
we'd just change this to `query: {openDataLoaderOverlay: true}` and check `props.location.query.openDataLoaderOverlay` in the `BucketList`